### PR TITLE
Update SDK with the new config update

### PR DESCRIPTION
**Pull request by Igoraamc**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [x] Other, please describe:
         Update config event.

**Does this PR introduce any breaking changes?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] Related issues are referenced in the PR (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] All tests are passing
- [x] New/updated tests are included

**Testing environment:**

- Operating System/Platform: macOS Catalina
- Node version: v12.16.0
- Yarn version: 1.22.0

If you have relevant logs, error output and etc, please attach them.

**Other information:**

### CODE REVIEW

**Review by asf89 — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -251,19 +251,24 @@ list of things:
...
 - `id` **String** Thing's ID.
-- `schema` **Array** Objects in the following format:
+- `config` **Array** Objects in the following format:
   * `sensorId` **Number** Sensor ID.
-  * `typeId` **Number** Semantic value type (voltage, current, temperature, etc).
-  * `valueType` **Number** Data value type (boolean, integer, etc).
-  * `unit` **Number** Sensor unit (V, A, W, W, etc).
-  * `name` **String** Sensor name.
-
+  * `schema` **JSON Object** Schema item, each one formed by:
```
Comment (esdrasjnr) [inline README.md:262]: I think that `Schema item, formed by` is enough. I think that with "each one" seems there is more than one schema object

File: README.md [hunk 2]
```diff
@@ -251,19 +251,24 @@ list of things:
...
+    * `typeId` **Number** Semantic value type (voltage, current, temperature, etc).
+    * `valueType` **Number** Data value type (boolean, integer, etc).
+    * `unit` **Number** Sensor unit (V, A, W, W, etc).
+    * `name` **String** Sensor name.
+  * `event` **JSON Object** Event item, each one formed by:
```
Comment (esdrasjnr) [inline README.md:267]: Same here

**Review by esdrasjnr — APPROVED** [review]
Looks great! 👍🏻 💯 Just left one little request, let me know if you agree 😉

File: README.md (hunk 2 above)
Comment (Igoraamc) [reply README.md]: Done!

File: README.md (hunk 1 above)
Comment (Igoraamc) [reply README.md]: Done!

**Review by netoax — CHANGES_REQUESTED** [review]
Add a commit bumping this package version to `3.0.0` and update the CHANGELOG with the changes description.

**Review by netoax — APPROVED** [review]
LGTM!

**Pull request merged**